compiler/rustc_mir_transform/src/add_retag.rs

```diff
@@ -34,7 +34,7 @@ fn is_stable(place: PlaceRef<'_>) -> bool {
 }
 
 /// Determine whether this type may be a reference (or box), and thus needs retagging.
-fn may_be_reference(ty: Ty<'tcx>) -> bool {
+fn may_be_reference<'tcx>(ty: Ty<'tcx>) -> bool {
```

Comment:
Looks like here and in some cases below `'_` can be used instead?

Reply:
I went through and replaced dozens of named lifetimes with `'_` 🙂